shell/common/api/electron_api_shell.cc

```diff
@@ -71,6 +71,12 @@ v8::Local<v8::Promise> OpenExternal(const GURL& url, gin::Arguments* args) {
   return handle;
 }
 
+void ShowItemInFolder(const base::FilePath& full_path) {
```

Comment:
it looks like this function [upstream](https://source.chromium.org/gn/gn/+/main:src/base/files/file_path.h;l=353?q=NormalizePathSeparators&sq=package:chromium&type=cs) does nothing on POSIX systems, so instead of adding this extra functionality here, let's instead call it directly [where relevant](https://github.com/electron/electron/blob/04b2ba84cd9c1c883498e0cee8492c4bb59088ec/shell/common/platform_util_win.cc#L335):

```diff
diff --git a/shell/common/platform_util_win.cc b/shell/common/platform_util_win.cc
index a2deae69bf..38bf94ab2e 100644
--- a/shell/common/platform_util_win.cc
+++ b/shell/common/platform_util_win.cc
@@ -336,7 +336,7 @@ void ShowItemInFolder(const base::FilePath& full_path) {
   base::ThreadPool::CreateCOMSTATaskRunner(
       {base::MayBlock(), base::TaskPriority::USER_BLOCKING})
       ->PostTask(FROM_HERE,
-                 base::BindOnce(&ShowItemInFolderOnWorkerThread, full_path));
+                 base::BindOnce(&ShowItemInFolderOnWorkerThread, full_path.NormalizePathSeparators()));
 }
 
 void OpenPath(const base::FilePath& full_path, OpenCallback callback) {
```